cloud/app/server/utils.js

```diff
@@ -105,11 +105,25 @@ const ensureCapabilityDB = async (capName) => {
     query: `GRANT ALL ON ${dbName}.* TO ${user}`
   });
 
+  // create row level security policy to allow access to all rows
+  await ClickHouse.client.exec({
```

Comment:
Why are we using `exec` instead of `query` or `command`?

From https://clickhouse.com/docs/integrations/javascript#exec-method:
> Exec method
> If you have a custom query that does not fit into query/insert, and you are interested in the result, you can use exec as an alternative to command.
> 
> exec returns a readable stream that MUST be consumed or destroyed on the application side.

I don't see us consume or destroy the readable stream. Probably not a big issue, but why not just use `command` instead which is for "when you are not interested in the response at all"?